src/main/controllers/cmc-service.js

```diff
@@ -56,8 +56,10 @@ module.exports = function (app) {
                         return callback(err);
                     });
                 }, function (err) {
-                    if (err) return log.error(err);
-                    log.info("TOKEN PRICE UPDATED");
+                    if (err) {
```

Comment:
This looks like debugging/development code. Do we need it in the production app?